reana.yaml

```diff
@@ -3,11 +3,12 @@ inputs:
   parameters:
     input: workflow/input.yaml
   directories:
-    - data
     - code
+    - data
+    - workflow/cwl
```

Comment:
There's no `workflow/cwl` path. We would need to arrange the files differently if we want to keep this.

Reply:
My bad, forgot to remove this when I changed the directory back